src/rdb.c

```diff
@@ -2343,7 +2343,6 @@ void startLoading(size_t size, int rdbflags) {
     server.loading_loaded_bytes = 0;
     server.loading_total_bytes = size;
     server.loading_rdb_used_mem = 0;
-    server.rdb_expired_keys_last_load = 0;
```

Comment:
>maybe we rather let this keep incrementing forever?
i.e. resetting it may be nicer as a single metric to look at with redis-cli.
not resetting it may be nicer for plotting this metric in some monitoring software (e.g. Grafana)

>if we don't reset it, we should obviously change the field name.

Need more suggestion, ping @redis/core-team 

BTW, I think we can add `empty_keys_skipped` `expired_keys_skipped` `keys_loaded` these three status in info reply.

Reply:
Ohh, sorry (for my commit), I noticed you moved the reset to resetServerStats, so I thought you agreed with me and just forgot the one in startLoading. 
I didn't notice the word "last" in the info field... 

Thinking of it again maybe you're right. 
We do have quite a few other fields named "last", and maybe it even gives better value to users than the cumulative stat. 

I was thinking of someone who plots the value over time in some stats system, so it's easy to see how much it got incremented on each loading phase. But would be less useful for a user typing INFO in redis-cli. And I suppose a value of a stat that's being reset on each loading is able to serve both purposes. 

So bottom line, I think the name can be changed to: `rdb_last_load_keys_expired`? 
I agree that it's also nice to add `rdb_last_load_keys_loaded`. 
I'm not sure about empty keys, in theory these should never exist, and we only print a warning to the log file, I don't consider that a real feature of Redis (justifying an info field)

Reply:
>So bottom line, I think the name can be changed to: rdb_last_load_keys_expired?
I agree that it's also nice to add rdb_last_load_keys_loaded.
I'm not sure about empty keys, in theory these should never exist, and we only print a warning to the log file, I don't consider that a real feature of Redis (justifying an info field)

Agree, let's rename and add the two fields.

BTW, IIUC we don't need to reset these in `resetServerStats()`.